layout-instability/idlharness.html

```diff
@@ -27,6 +27,16 @@
       return;
     }
 
+    // Make sure that the image has initially been laid out, so that the movement
```

Comment:
nit: maybe use await waitForAnimationFrames()? This is defined in resources/util.js.